lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Uuid.php

```diff
@@ -25,7 +25,7 @@
  * @Annotation
  * @Target("PROPERTY")
  */
-final class Uuid extends Property
+final class Uuid extends BaseProperty
```

Comment:
if somebody was typechcking Property in custom code handling annotations, this is a BC break. do we need to worry?